### PR TITLE
1871 add debug to docs

### PR DESCRIPTION
**Pull request by Tthomas63**
Adds a debugging page to the `readthedocs` detailing several useful docker, database, and python commands.

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| `create_competition`     | Creates a competition   | `--title` title, `--description` description, `--logo` logo file for comp, `--force_user_create` create user if non-existant, `--creator` creator email, `--numphases` number of phases, `--phasedates` Comma-seprated list of the startdates (YYYY-MM-DD, YYYY-MM-DD) | `create_competition ..` |
+| `create_solution`        | Creates a submission    | `d` don't delete temp files, `-w` Wrong. Makes a submission that fails.| `create_solution -d -w`|
+| `create_codalab_user`    | Creates a user          | `password`                         | `create_codalab_user "bongo"`     |
+
+## Database
+
+Sometimes it is necessary to destroy, and re-create the database. In those cases use the following commands from
+a bash terminal inside the database docker container. IE: `docker exec -it postgres bash`
+
+`dropdb <db_name>`
```
Comment (ckcollab) [inline docs/Debugging.md]: change this to ```drop <db_name>```

Comment (ckcollab) [inline docs/Debugging.md]: 3 ticks instead of 1, so ```

File: docs/Debugging.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+ Removes the database with `<db_name>`
+
+`createdb <db_name>`
```
Comment (ckcollab) [inline docs/Debugging.md]: same as above

File: docs/Debugging.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Debugging
+These are several useful commands you may find useful for debugging.
```
Comment (ckcollab) [inline docs/Debugging.md]: maybe add a newline after this

File: docs/Debugging.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+| `create_competition`     | Creates a competition   | `--title` title, `--description` description, `--logo` logo file for comp, `--force_user_create` create user if non-existant, `--creator` creator email, `--numphases` number of phases, `--phasedates` Comma-seprated list of the startdates ```(YYYY-MM-DD, YYYY-MM-DD)``` | ```create_competition ..``` |
+| `create_solution`        | Creates a submission    | `d` don't delete temp files, `-w` Wrong. Makes a submission that fails.| `create_solution -d -w`|
+| `create_codalab_user`    | Creates a user          | `password`                         | `create_codalab_user "bongo"`     |
+
+## Database
+
+Sometimes it is necessary to destroy, and re-create the database. In those cases use the following commands from
+a bash terminal inside the database docker container. IE: ```docker exec -it postgres bash```
+
+```
```
Comment (ckcollab) [inline docs/Debugging.md]:
```
`-``drop <db_name>`-``

Removes the database with `<db_name>`

`-``create <db_name>`-``

Creates a database with `<db_name>`

```

Replace `-`` with 3 ticks

File: docs/Debugging.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+# Debugging
+
+These are several useful commands you may find helpful for debugging.
```
Comment (ckcollab) [inline docs/Debugging.md]: newline after this sentence?

**Comment by ckcollab** [comment]
Goahead and merge! :shipit:

**Comment by Tthomas63** [comment]
Not authorized 😛

**Pull request merged**